samples/excel/89-preview-apis/events-tablecollection-changed.yaml

```diff
@@ -0,0 +1,164 @@
+order: 9
+id: excel-events-tablecollection-changed
+name: Events - Table collection changed
+description: Add event handlers for table collection onDataChanged event
```

Comment:
If it's just one event handler, suggest you change "event handlers" to "event handler".